docs/our_work/epma_autocoding.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'ePMA Auto-Coding'
+summary: 'A rule-base data processing pipeline to map free text medication descriptions to SNOMED codes using the dm+d standard.'
+origin: ''
+tags: ['PYTHON','SECONDARY CARE','PRESCRIBING','TEXT DATA']
+---
+## The Problem
+The ePMA (electronic prescribing and medicine administration) data collection is a person-level data asset containing information on what medication is prescribed and administered in secondary care settings. The ePMA data that hospitals submit contain a number of free text fields including: medication name, dosage, route and frequency. This creates some challenges when aiming to collect and eventually disseminate this data:
+
+1. Free text can potentially contain PID (patient identifiable data) which is not permitted
+2. Free text descriptions of medication makes analysis hard because the same medication could be described in multiple ways 
+
```

Comment:
gramatically this should either be: 
```suggestion
2. Free text descriptions of medication make analysis hard because the same medication could be described in multiple ways"

```

or

```suggestion
2. Free text description of medication makes analysis hard because the same medication could be described in multiple ways"

```